workers/data_refinery_workers/processors/test_utils.py

```diff
@@ -3,161 +3,49 @@
 from unittest.mock import MagicMock
 from django.test import TestCase
 from data_refinery_common.models import (
+    OriginalFile,
+    ProcessorJobOriginalFileAssociation,
     SurveyJob,
-    Batch,
-    BatchStatuses,
-    File,
     ProcessorJob,
 )
 from data_refinery_workers.processors import utils
 
+def prepare_job():
+    pj = ProcessorJob()
+    pj.pipeline_applied = "AFFY_TO_PCL"
+    pj.save()
 
-def init_objects():
-    survey_job = SurveyJob(
-        source_type="ARRAY_EXPRESS"
-    )
-    survey_job.save()
+    og_file = OriginalFile()
```

Comment:
I guess the term `OG File` you used earlier means "Original File"? Do you mind using a more explicit variable name?